.github/workflows/ci.yml

```diff
@@ -31,6 +31,10 @@ jobs:
       run: |
         clang main.c -o ruapu-clang
         ./ruapu-clang
+    - name: build-test-python
+      run: |
+        pip3 install .
+        python3 -c 'import ruapu; print(ruapu.supports("neon")); print(ruapu.supports(isa="avx"))'
```

Comment:
ci log shows that an UNKNOWN package was built
try upgrading pip or setuptools ?

```
Building wheels for collected packages: UNKNOWN
  Building wheel for UNKNOWN (pyproject.toml): started
  Building wheel for UNKNOWN (pyproject.toml): finished with status 'done'
  Created wheel for UNKNOWN: filename=UNKNOWN-0.0.0-cp310-cp310-linux_x86_64.whl size=13600 sha256=8084b088ef4675e5144eb1b6fe5e592eebf3dc1744ed0a511e9613562394fd44
  Stored in directory: /tmp/pip-ephem-wheel-cache-smr2q52j/wheels/6e/d5/b4/854280212ff45b045af166a62e2eeac73e7a772e78fd439104
Successfully built UNKNOWN
Installing collected packages: UNKNOWN
Successfully installed UNKNOWN-0.0.0
```

Reply:
Updating setuptools seems to solve it. ;)